C++/Graphs/Kruskal_Algo.cpp

```diff
@@ -0,0 +1,71 @@
+// this algorith uses DSU to make MST
+// step 1: sort all edges according to their weights
+// step 2: iterate over all edges and check if u and v don't belong to the same component, join them
+
+#include <bits/stdc++.h>
+using namespace std;
+
+struct node {
+    int u, v, wt;
+    node(int first, int second, int weight) {
+        u = first;
+        v = second;
+        wt = weight;
+    }
+};
+
+bool comp(node a, node b) {
+    return a.wt < b.wt;
+}
+
+int findPar(int node, vector<int> &parent) {
+    if (node == parent[node]) return node;
+    return parent[node] = findPar(parent[node], parent);
+}
+
+void unionn(int u, int v, vector<int> &parent, vector<int> &Rank) {
+    u = findPar(u, parent);
```

Comment:
Try to add a comment here for eg why we are finding out parents? How it would help ?